### DISCUSSION TITLE
Extract and Diff dialog title change?

### DISCUSSION THREAD

**Discussion by delfino-nunez** [borgbase/vorta#1243] [Ideas]
Should the Extract and Diff dialogs have different titles instead of the generic "dialog"?

Extract:
![Dialog_001](https://user-images.githubusercontent.com/770967/160532712-d3237e46-86f1-47c4-b35f-fcff656adc51.png)

Diff:
![Dialog_003](https://user-images.githubusercontent.com/770967/160533237-0d55f8f1-9db3-4fa9-acf8-8453902ee534.png)

Diff Result:
![Dialog_002](https://user-images.githubusercontent.com/770967/160532733-b203e0d5-3123-408e-8084-91552a373dca.png)

**Comment by real-yfprojects** [comment]
I think so. Maybe something like this:

Extract: Choose files to extract
Diff: Choose archives for diff
Diff Result: Diff of {archive1} and {archive2}

**Reply by delfino-nunez** [reply]
The first 2 look great.

For Diff Result I would leave the title just to "Diff Result" since there's already a label with the 2 archives compared. Also I wonder if the archive names are too long the title will not show correctly.

**Reply by real-yfprojects** [reply]
Let's use "Diff Result" then.

**Comment by delfino-nunez** [comment]
This was merged on #1246
